content/guides/autodiscovery.md

```diff
@@ -48,155 +86,169 @@ By default, the Datadog Agent includes Autodiscovery support for:
 
-    docker_images:
-      - httpd
+Follow the [Kubernetes integration instructions](/integrations/kubernetes/), then add annotations to your pod definitions. Autodiscovery expects annotations to look like the following:
 
-    init_config:
+~~~
+annotations:
+  service-discovery.datadoghq.com/<Kubernetes Container Name>.check_names: '[<CHECK_NAME>]'
+  service-discovery.datadoghq.com/<Kubernetes Container Name>.init_configs: '[<INIT_CONFIG>]'
+  service-discovery.datadoghq.com/<Kubernetes Container Name>.instances: '[<INSTANCE_CONFIG>]'
+~~~
 
-    instances:
-      - apache_status_url: http://%%host%%/server-status?auto
+It's very similar to the template format for key-value stores. Autodiscovery looks for annotation keys beginning with `service-discovery.datadoghq.com/`. Kubernetes container names function as the service identifiers.
 
-To store the same configuration template in etcd you could run the following commands:
+The following pod annotation defines two templates—equivalent to those from the end of the previous section—for `apache` containers:
 
-    etcdctl mkdir /datadog/check_configs/httpd
-    etcdctl set /datadog/check_configs/httpd/check_names '["apache"]'
-    etcdctl set /datadog/check_configs/httpd/init_configs '[{}]'
-    etcdctl set /datadog/check_configs/httpd/instances '[{"apache_status_url": "http://%%host%%/server-status?auto"}]'
+~~~
+apiVersion: v1
+kind: Pod
+metadata:
+  name: apache
+  annotations:
+    service-discovery.datadoghq.com/apache.check_names: '["apache","http_check"]'
+    service-discovery.datadoghq.com/apache.init_configs: '[{},{}]'
+    service-discovery.datadoghq.com/apache.instances: '[{"apache_status_url": "http://%%host%%/server-status?auto"},{"name": "My service", "url": "http://%%host%%", timeout: 1}]'
+  labels:
+    name: apache
+spec:
+  containers:
+    - name: apache
+      image: httpd
+      ports:
+        - containerPort: 80
+~~~
 
-### Image name format in the configuration store
+# Detailed examples (i.e. Case studies)
+
+(Only in blog post?)
+
+# Reference
+
+### Template Variable Indexes
+
+For containers that have many IP addresses or listens on many ports, you can tell Autodiscovery which ones to choose by appending an underscore to the template variable, followed by an index, e.g. `%%host_0%%`, `%%port_4%%`. After inspecting the container, Autodiscovery sorts the IPs and ports **numerically and in ascending order**. For a container that listens on ports 80, 443, and 8443, `%%port_0%%` refers to port 80. Unindexed template variables refer to the last item in the sorted list, so in this case, `%%port%%` means port 8443.
+
+You can also add a network name suffix to the `%%host%%` variable—`%%host_bridge%%`, `%%host_swarm%%`, etc—for containers attached to multiple networks. When `%%host%%` does not have a suffix, Autodiscovery picks the container's bridge network IP address.
+
+### Service Identifiers
```

Comment:
Well, I had two support cases that could've been solved by using container labels instead of toying around with the image names, so I'd vote for pushing it up. But that's probably sampling biais